server/models/video/video-file.ts

```diff
@@ -157,17 +194,20 @@ export class VideoFileModel extends Model<VideoFileModel> {
               .then(results => results.length === 1)
   }
 
-  static loadWithVideo (id: number) {
-    const options = {
-      include: [
-        {
-          model: VideoModel.unscoped(),
-          required: true
-        }
-      ]
-    }
+  static async doesVideoExistForVideoFile (id: number, videoIdOrUUID: number | string) {
+    const videoFile = await VideoFileModel.loadWithVideo(id)
```

Comment:
`loadWithVideo` does not load the video file with the streaming playlist video. Please make a SQL request 

```
include: [
        {
          model: video model
          required: false,
          where { uuid or id: ... }
        },
        {
          model: streaming playlist model,
          required: false,
          include: [
                    { model: video model, required: true, where { uuid or id ... } }
          ]
      ],
```